src/engraving/dom/engravingitem.cpp

```diff
@@ -103,6 +103,8 @@ EngravingItem::EngravingItem(const EngravingItem& e)
     m_minDistance = e.m_minDistance;
     itemDiscovered = false;
 
+    setAutoplace(e.autoplace());
```

Comment:
It seems to me that this would not have any effect:
- the autoplace value is stored in `m_flags`, which is properly copied from the other element
- I believe calling virtual methods in a constructor does not work, i.e. in this case just `EngravingItem::setAutoplace` is called (because the VTable is not fully initialised at this point (?)); 
That is relevant because `EngravingItem::setAutoplace` only sets the `m_flags` flag, which, by the previous point, should already have the correct value. 

If it does have some effect in practice, I would be interested why. 


(By the way, about calling virtual methods inside a constructor: I'm not sure if that is just how it works, or that doing it even invokes "Undefined Behaviour", i.e. in the C++ standard it's not specified what should happen, so nobody knows what will happen.)

Reply:
Good point, I may have tried that initially and then not realized I could remove it. I'll check 👍